dbt_meshify/storage/dbt_project_editors.py

```diff
@@ -85,14 +103,29 @@ def _get_subproject_boundary_models(self) -> Set[str]:
         )
         return boundary_models
 
+    def create_starter_project(self) -> DirectoryChange:
+        """Create a new starter project using the default stored in dbt-core"""
+
+        return DirectoryChange(
+            operation=Operation.Copy,
+            entity_type=EntityType.Directory,
+            identifier=str(self.subproject.path),
+            path=self.subproject.path,
+            source=get_starter_project_path(),
+            ignore_function=shutil.ignore_patterns("__init__.py", "__pycache__", "**/*.pyc"),
+        )
+
     def write_project_file(self) -> FileChange:
         """
         Writes the dbt_project.yml file for the subproject in the specified subdirectory
         """
+
+        # Read a starter `dbt_project.yml` file as a baseline
+        starter_path: Path = get_starter_project_path() / "dbt_project.yml"
+        starter_dbt_project = YAMLFileManager.read_file(starter_path)
+
         contents = self.subproject.project.to_dict()
-        # was getting a weird serialization error from ruamel on this value
-        # it's been deprecated, so no reason to keep it
-        contents.pop("version")
+
         # this one appears in the project yml, but i don't think it should be written
         contents.pop("query-comment")
```

Comment:
i remember this line being pretty /shrug at the time i wrote it -- any chance you tested removing this line and seeing if it worked? I feel like it may not in be in the yml from the starter project, and therefore not something we'd need to worry about anymore.

Reply:
Based on my tests, if we remove it from here, it will be rendered in the sub-project's `dbt_project.yml`. Is that the desired behavior?

Reply:
no i wouldn't say so 😅 i think my thought was that it may no longer be in the YML we start with now that we're using the official starter project file. if it's still there, I'd vote not to render it